3_visualize/src/plot_output_weights.py

```diff
@@ -0,0 +1,57 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:percent
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.13.7
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %%
+import sys
+import numpy as np
+import matplotlib.pyplot as plt
+
+# %%
+sys.path.insert(0, "../../2a_model/src/models/0_baseline_LSTM/")
+
+# %%
+from model import LSTMModel
+
+# %%
+m = LSTMModel(10, 3)
+
+# %%
+m.load_weights("../../2a_model/out/models/0_baseline_LSTM/train_weights/")
```

Comment:
I'm a little confused where these weights come from

Reply:
These are weights produced by the snakemake workflow:
https://github.com/USGS-R/drb-do-ml/blob/cee3da5b747c035c826641e9bdbb05f91b394a81/2a_model/src/models/0_baseline_LSTM/Snakefile#L71